### PR TITLE
Support get/set.

### LINKED ISSUE #54: Support get and set methods
Error: modules/angular2/src/di/key.js:27:3: Unsupported node type GetAccessor:

  static get numberOfKeys():int {
    return _globalKeyRegistry.numberOfKeys;
  }

### PR DESCRIPTION
**Pull request by alexeagle**
Fixes #54

### CODE REVIEW

File: main.ts [hunk 1]
```diff
@@ -41,8 +41,11 @@ class Translator {
     this.emit(')');
   }
 
-  visitFunctionLike(fn: ts.FunctionLikeDeclaration) {
-    this.visitParameters(fn);
+  visitFunctionLike(fn: ts.FunctionLikeDeclaration, accessor: string = null) {
```
Comment (mprobst) [inline main.ts]: wouldn't this better be `accessor?: string` ?

File: main.ts [hunk 2]
```diff
@@ -41,8 +41,11 @@ class Translator {
...
+    if (fn.type) this.visit(fn.type);
+    if (accessor) this.emit(accessor);
+    if (fn.name) this.visit(fn.name);
+    if (accessor !== 'get') this.visitParameters(fn);
```
Comment (mprobst) [inline main.ts]: Can you really not have parameters in a getter? Who enforces that? Maybe it's better to error out or just generate the code and have the Dart compiler error out?

File: main.ts (hunk 1 above)
Comment (alexeagle) [inline main.ts]: yes! learning TypeScript...

File: main.ts (hunk 2 above)
Comment (alexeagle) [inline main.ts]: Added a comment and also validate that getter has no params

File: test/e2e/lib.ts [hunk 3]
```diff
@@ -1,14 +1,14 @@
 class MyClass {
-  field: string;
+  _field: string;
```
Comment (mprobst) [inline test/e2e/lib.ts]: FYI you'll need to put private in front of this.

**Comment by mprobst** [comment]
LGTM.

File: test/e2e/lib.ts (hunk 3 above)
Comment (alexeagle) [inline test/e2e/lib.ts]: Unsupported node type PrivateKeyword
maybe you just mean later when you merge some more changes

Comment (alexeagle) [inline test/e2e/lib.ts]: strange, rebase claimed I was up to date but pull brought in conflicts...

**Pull request merged**